src/language/grammar.tx

```diff
@@ -32,5 +34,10 @@ Is_Close_To_Con: 'distClose' '(' actors += [Actor] ',' actors += [Actor] ')' ;
 Is_Medium_Distance_From_Con: 'distMed' '(' actors += [Actor] ',' actors += [Actor] ')' ;
 Is_Far_From_Con: 'distFar' '(' actors += [Actor] ',' actors += [Actor] ')' ;
 
+Behavior_Con: Does_Maneuver_Con | Danger_Con ;
+Danger_Con: Collision_Con ;
+
+Does_Maneuver_Con: 'doesManeuver' '(' actors += [Actor] ',' maneuver = ID ')' ;
```

Comment:
Aggregation strategy by actors (src/search/mhs/utils.py line 70) expects a field called "actors", so I made it a list and named it "actors" even if there is just one actor

Reply:
Ah, I see. This is a corner case, because in the MHS mode, the 'maneuverJunction' is not treated as a regular constraint. It is more like an attribute of the Scenario Specification. 

tbd

Reply:
Actually, what I mean here is just that there would be just one actor but I named it `actors` just to be consistent.

But yes, for `doesManeuver` (and also `onRegion`), the maneuver/region will be an extra field